torntools/scripts/content/displaycase/ttDisplayCase.js

```diff
@@ -0,0 +1,27 @@
+requireDatabase().then(() => {
+	console.log("TT - Display Case");
+	displayCabinetWorth();
+});
+
+function displayCabinetWorth() {
+	if (settings.pages.displaycase.worth) {
+		let display_user_id = window.location.href.split("/")[4];
```

Comment:
It would be best to use the formatting defined in the CONTRIBUTING.md:

> * We also use some standard formatting conventions.
>   * Identifier names (variables and functions) start with a letter and use camelCase.
>     * If it are global constants use UPPERCASE.

I do know that our current code is not consistent in this, but new code should follow these guidelines.

Reply:
I am sorry but this code is copied from ttBazaar.js and modified it according to the needs. Changed here.

Reply:
Yeah, it's old code from when we didn't have a CONTRIBUTING.md, or probably even before I joined the team. And since it worked, I didn't go through it all just to change some names.